src/main/java/uk/gov/hmcts/reform/orgrolemapping/domain/model/FlagRequest.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.hmcts.reform.orgrolemapping.domain.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Value;
+
+@Value
+@Builder
+@AllArgsConstructor
+@Getter
```

Comment:
No Need of @AllArgsConstructor and @Getter. As, @Value is the combination of these.

Reply:
Done